zephyr/lc3_config.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2022 Bose Corporation All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include "Lc3Config.hpp"
+#include "lc3_config.h"
+
+Lc3Config_t Lc3Config_create(uint16_t Fs_, lc3_frame_duration_t N_ms_, uint8_t Nc_)
+{
+    Lc3Config* inst;
+    Lc3Config::FrameDuration duration;
+    switch(N_ms_)
+    {
+    case lc3_duration_7p5ms:
+        duration = Lc3Config::FrameDuration::d7p5ms;
+        break;
+    case lc3_duration_10ms:
+        duration = Lc3Config::FrameDuration::d10ms;
+        break;
+    }
```

Comment:
When doing a switch on an enum some compilers complain if there is a default case when all enum values have already been handled. Should I add a dummy value to the enum to avoid this?

Reply:
Oh, didn't know that. 

In that case I am not sure what the correct solution is :D If the code is implemented in a specific way for one or more specific compilers, that is seem like a slippery slope. Our source code should be compiler agnostic. I bet someone (perhaps @carlescufi ?) know a lot more about this.